qiskit_ibm_runtime/estimator.py

```diff
@@ -104,16 +107,17 @@ class EstimatorV2(BasePrimitiveV2[EstimatorOptions], Estimator, BaseEstimatorV2)
 
     def __init__(
         self,
-        backend: Optional[Union[str, IBMBackend]] = None,
+        mode: Optional[Union[IBMBackend, Session, Batch]] = None,
+        backend: Optional[IBMBackend] = None,
         session: Optional[Session] = None,
         options: Optional[Union[Dict, EstimatorOptions]] = None,
     ):
         """Initializes the Estimator primitive.
 
         Args:
-            backend: Backend to run the primitive. This can be a backend name or an :class:`IBMBackend`
-                instance. If a name is specified, the default account (e.g. ``QiskitRuntimeService()``)
-                is used.
+            mode: Session or Batch in which to call the primitive.
```

Comment:
We should give people more information since this is somewhat new, e.g. 

```
    mode: The execution mode used to make the primitive query. It can be
        
        * A :class:`Backend` if you are using job mode. 
        * A :class:`Session` if you are using session execution mode.
        * A :class:`Batch` if you are using batch execution mode. 
        
        For more information about the execution modes, prefer to <link> 
```